aladdin/lib/aws/dns_mapping.py

```diff
@@ -2,11 +2,14 @@
 import logging
 from datetime import datetime
 
+from aladdin.lib.cluster_rules import ClusterRules
+
 
 def fill_hostedzone(
-    boto_session, cluster_domain_name, namespace_domain_name, hostnames_to_loadbalancers
+    boto_session, hostnames_to_loadbalancers, cluster_domain_name=None, namespace_domain_name=None
 ):
-    # Main DNS is on prod, sub DNS should be on sandbox
+    cluster_domain_name = cluster_domain_name or ClusterRules().cluster_domain_name
+    namespace_domain_name = namespace_domain_name or ClusterRules().namespace_domain_name
```

Comment:
the singleton stuff above is mainly to support this right here, in case the caller does not provide these domain names